### PR TITLE
Added documentation

### PR DESCRIPTION
**Pull request by vinitshahdeo**
Changes:

- Added report by @ishika1727
- YouTube Video by @ishika1727

### CODE REVIEW

**Comment by vinitshahdeo** [comment]
@ishika1727 Please review this. I have improved `documentation.md` a bit.

**Comment by ishika1727** [comment]
The changes look great 👍  @vinitshahdeo

**Comment by vinitshahdeo** [comment]
@Kashish121 Changes look good to you? 
> **Waiting for your approval!**

File: docs/documentation.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+With this port scanner, I just attempt to connect at various ports. If I'm able to connect to open ports, then I know at least the port is open. The code is in python and the server is built using express.js. 
+
+
+## Working
+
+The server code is executed first where it opens 10 random ports (random numbers generated by `randomNum.js`) and the `index.js` starts ten servers. Then the `scanner.py` searches for all the open ports on remote server using sockets (socket library of python). The `index.js` is coded in express which is a server side scripting language and creates and starts servers. The scanning code for open ports is build using python programming language. The `scanner.py` code imports and uses several python libraries like socket, subprocess and sys. 
+`index.js` imports randNum.js to generate random number and open ports at those `localhost`. It also imports config.json which stores values of low and high i.e. values within whose range the ports will open. In this code, it is set from 1 (inclusive) to 80 (exclusive). The code generates 10 random numbers from 1 to 80 i.e low to high and starts servers at those ports.
```
Comment (Kashish121) [inline docs/documentation.md:14]: The code generates 10 random numbers from **1 to 80** i.e low to high and starts servers at those ports. 

Should be changed to

The code generates 10 random numbers from **1 to 8888** i.e low to high and starts servers at those ports.

**Comment by vinitshahdeo** [comment]
@Kashish121 Review now, updated the PR.

**Pull request merged**